docker-compose-files/dspace-compose/ComposeFiles.md

```diff
@@ -0,0 +1,74 @@
+## Proposed Changes to Docker Compose Startup
+
+| Compose File | Purpose |
+| -- | -- |
+| docker-compose.yml | Base image for DSpace 4, 5, 6, 7.  Ingests content (if needed) and starts tomcat. |
+| d4.override.yml | Sets base image for DSpace4x.  <br/>Since Flyway is not supported in DSpace4x, a special 4x postgres image. |
+| d5.override.yml | Sets base image for DSpace5x.  Activates Mirage2. |
+| d6.override.yml | Sets base image for DSpace6x.  Activates Mirage2. |
+| d7.override.yml | Sets base image for DSpace7x.  <br/>Adds Angular UI to replace XMLUI and JSPUI. <br/>TBD: Add externalized SOLR service<br/> REST: http://localhost:8080/spring-rest <br/> Angular UI: http://localhost:3000 |
+| src.override.yml | Optional add-on to trigger and redeploy and tomcat start. |
+| rdf.override.yml | Optional RDF Add-on for DSpace6x and DSpace7x. <br/>http://localhost:3030 |
+
+## Basic Startup Commands
+
+```
+docker-compose -p d7 -f docker-compose.yml -f d7.override.yml up -d
```

Comment:
If I run all of these commands one after the other, everything goes well but then I receive an error regarding the shared `/dspacedb`:

> ERROR: for dspacedb  Cannot create container for service dspacedb: Conflict. The container name "/dspacedb" is already in use by container "c8bb39ef47acadff2eb13d618c8a170b63c8781bac8abadd38157072fb4d1bf4". You have to remove (or rename) that container to be able to reuse that name.

If I run the `down` argument, as described below, it solves the issue, perhaps these commands could be separated so that it makes it more clear that they should be run one at a time.

Reply:
@PTrottier , I added a commit with some clarifying notes about these commands.  Let me know if this resolves the issue for you.  Thanks again for the review!